grapher/color/ColorScale.ts

```diff
@@ -215,7 +215,7 @@ export class ColorScale {
     }
 
     // Exclude any major outliers for legend calculation (they will be relegated to open-ended bins)
-    @computed private get sortedNumericValuesWithoutOutliers() {
```

Comment:
I opted for making this method public to be able to do it directly, even if we should probably mostly test public-facing methods normally. But I found it way easier to do it this way, other ways would have been way more indirect.

Reply:
I do this a lot too. Actually not sure what the best practice is here. Something for us to read up on at some point.

Reply:
I think the common best practice is that only public-facing APIs should be tested (so as to not rely on implementation details), but I still think it's worthwhile to also test private methods if that makes the tests clearer.

In this cases, testing the output bins would be many levels removed from the outlier detection itself, and it would be hard to see the actual intended behavior from just looking at the test cases.

Reply:
Yeah you know what, perhaps when we have a situation like this the thing to do is make it a util method and test that. Maybe that is the common pattern (I've got 12+ methods I made public for this reason, I should take a look and see if that pattern holds). Nothing to do now, just musing aloud.